virttest/nfs.py

```diff
@@ -203,10 +203,9 @@ def setup(self):
         service and exportfs too.
         """
         if self.nfs_setup:
-            if not self.nfs_service.status():
```

Comment:
remove the judgment is not good choice.
you should find out the cause leads to the failure of umount operation.